README.md

```diff
@@ -229,8 +229,8 @@ const config = {
       {
         test: /\.html$/,
         use: [
-          { loader: 'html-loader', options: { minimize: true } }
-          'posthtml-loader'
+          { loader: 'html-loader', options: { minimize: true } },
+          { loader: 'posthtml-loader' }
```

Comment:
Thanks for your reply :) 
anyway, about the example of config of webpack2, there are some problems, could you please help me to figure them out?